README.md

```diff
@@ -401,6 +401,11 @@ const api = new Gitlab({
 // Listing users
 let users = await api.Users.all();
 
+// Listing project members(including inherited members)
+// see. https://docs.gitlab.com/ee/api/members.html#list-all-members-of-a-group-or-project-including-inherited-members
+const projectId = 9999;
```

Comment:
does not have to be a projectId could just as well be the group/project ;)

Reply:
Sorry , I don't understand well ... 

Should I add sample code that uses ```GroupMembers```? or elase magic number ```projectId```?

ex.
```
// Or using Promise-Then notation
api.Projects.all().then((projects) => {
  console.log(projects);
  // add sample codes.
});
```

Reply:
For the example I do not think project id is helpful.
Yes you can use project id if you know them by heard or doing a sequence of nested api calls like first getting projects and than using those project to.

My suggestion for the example is to the namespace path so `group/project` or `gitlab-org/gitlab` just like in the url to the project.
https://gitlab.com/gitlab-org/gitlab

Reply:
Thank you for pointing this out.
Updated sample code.